hxd/System.hl.hx

```diff
@@ -419,15 +419,17 @@ class System {
 		#if !usesys
 		hl.Api.setErrorHandler(function(e) reportError(e)); // initialization error
 		sentinel = new hl.UI.Sentinel(30, function() throw "Program timeout (infinite loop?)");
-		haxe.MainLoop.add(timeoutTick, -1) #if (haxe_ver >= 4) .isBlocking = false #end;
 		#end
 		#if ( target.threaded && (haxe_ver >= 4.2) )
 		mainThread = sys.thread.Thread.current();
 		#end
 	}
 	
 	#if (hlsdl || hldx)
-	static var _ = haxe.MainLoop.add(updateCursor, -1) #if (haxe_ver >= 4) .isBlocking = false #end;
+	static var _ = {

```

Comment:
Shouldn't this have `@:keep` in case of `--dce full`?

Reply:
Uh oh! True, we need `@:keep` here